aidb/engine/base_engine.py

```diff
@@ -294,21 +294,24 @@ def add_filter_key_into_query(
     param query_expression: sqlglot parsed query expression
     '''
     selected_column = []
-    col_name_set = set()
+    col_name_list = []
```

Comment:
Why was this changed to a list?

Reply:
Because choosing certain row in pd.Dataframes requires a list format. And in line 304, we have a condition to avoid duplicate, so the logic doesn't change.
https://github.com/ddkang/aidb/blob/f556532746d97f85d170172a974261af6d00cc2e/aidb/engine/base_engine.py#L304